JitsiConference.js

```diff
@@ -1273,9 +1273,11 @@ function setupListeners(conference) {
                     session, conference.settings);
             });
 
-        conference.room.addListener(XMPPEvents.CONFERENCE_SETUP_FAILED,
-            function () {
-                conference.statistics.sendSetupFailedEvent();
+        conference.room.addListener(XMPPEvents.CONNECTION_ICE_FAILED,
```

Comment:
Doesn't that mean that we aren't going to log the other cases when we receive XMPPEvents.CONFERENCE_SETUP_FAILED?